helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -91,6 +91,22 @@ spec:
         cancel_if_outside_working_hours: "true"
         severity: page
         topic: releng
+    - alert: ClusterCapiAppFailed
+      annotations:
+        description: '{{`Cluster CAPI App {{ if $labels.exported_namespace }}{{ $labels.exported_namespace }}{{ else }}{{ $labels.namespace }}{{ end }}/{{ $labels.name }}, version {{ $labels.version }} is in {{ $labels.status }} state.`}}'
+        opsrecipe: app-failed/
+      # we have this Frankenstein for namespace checks because go (and prometheus) don't support negative lookaheads in regex
```

Comment:
It looks like this comment doesn't apply anymore, as it has been extracted into `namespaceNotGiantSwarm`.